### PR TITLE
143 Support more than one cosignatory (multisigmodification)

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/commands/transaction/multisigmodification.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class CommandOptions extends AnnounceAggregateTransactionsOptions {
 
     @option({
         flag: 'p',
-        description: 'Cosignatory account public key.',
+        description: 'Cosignatory account public key (separated by a comma).',
         validator: new PublicKeyValidator(),
```
Comment (dgarcia360) [inline src/commands/transaction/multisigmodification.ts]: todo: create``PublicKeysValidator`` to support multiple keys validation.

File: src/commands/transaction/multisigmodification.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ export default class extends AnnounceAggregateTransactionsCommand {
         options.cosignatoryPublicKey = OptionsResolver(options,
             'cosignatoryPublicKey',
             () => undefined,
-            'Introduce the cosignatory account public key: ');
+            'Introduce the cosignatory account public key. (separated by a comma): ');
```
Comment (dgarcia360) [inline src/commands/transaction/multisigmodification.ts]: Remove dot ``...public key (separated by a comma):...``

File: src/commands/transaction/multisigmodification.ts (hunk 1 above)
Comment (NemTechCN) [reply src/commands/transaction/multisigmodification.ts]: fixed

File: src/commands/transaction/multisigmodification.ts (hunk 2 above)
Comment (NemTechCN) [reply src/commands/transaction/multisigmodification.ts]: fixed

**Comment by dgarcia360** [comment]
Closes #143

**Pull request merged**